3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -20,12 +20,21 @@ const rollDice = require('../../helpers/pokerDiceRoller');
 function rollTheDices() {
   // TODO Refactor this function
   const dices = [1, 2, 3, 4, 5];
-  return rollDice(1);
+  const resultPromises = [];
+  dices.forEach((dice) => {
+    resultPromises.push(rollDice(dice));
+  });
+  return resultPromises;
```

Comment:
Can you try to use `.map()` here instead of `.forEach()`? Map dice number to promises? And then return the return the result of `Promise.all()` (itself a promise) as the return value of this function. With those changes, you do not need to modify line 30 from the way it was.